### ISSUE TITLE
Adding `note` to ${bookmarkYaml}

### ISSUE THREAD

**Issue by flamedfury** [katydecorah/seasonal-post-action#133]
Hey Katy, me again.

Where is it configured for what gets passed to `${bookmarkYaml}` for the seasonal post template? I'd love to have the `note` and even `quote` available to include these in my seasonal post.

Any chance you could make this happen? :)

**Comment by katydecorah** [comment]
Yeah, sounds cool! I might need to rethink the authoring experience a bit to make this happen..... I'm on it 😀
